src/matcha_ml/infrastructure/output.tf

```diff
@@ -56,5 +56,16 @@ output "model_deployer_seldon_base_url" {
 }
 
 output "cloud_azure_resource_group_name" {
+  description = "The full name of the resource group"
   value = module.resource_group.name
 }
+
+output "cloud_azure_prefix" {
```

Comment:
I don't think these are required, as we are already getting and updating these values on line 185 of `azure_template.py`